docs/network-policy.md

```diff
@@ -345,3 +346,10 @@ the policies that may affect their workloads.
 - The v1alpha1 Policy CRDs support up to 10000 unique priority at policy level.
   In order to reduce the churn in the agent, it is recommended to set the
   priority within the range 1.0 to 100.0.
+
+## Known Issues
+
+- Creating an Antrea NetworkPolicy with the same name as a K8s NetworkPolicy
+  under the same Namespace is known to collide with each other and cause
+  unexpected outcome. See issue [1173](https://github.com/vmware-tanzu/antrea/issues/1173)
```

Comment:
Not grammatically correct. Suggest one of:
 * An Antrea NetworkPolicy and a K8s NetworkPolicy with the same name and under the same Namespace are known to collide with each other and cause unexpected behavior
 * Creating an Antrea NetworkPolicy with the same name as a K8s NetworkPolicy under the same Namespace creates a collision and causes unexpected behavior

Reply:
updated with 2nd suggestion